### ISSUE TITLE
Trouble with eslint 1.4.1 : no error reported

### ISSUE THREAD

**Issue by obrac** [eslint/eslint#3783]
Hi,

After a fresh install of eslint 1.4.1, I have tested it with this `test.js` file in a new directory:
```js
var foo = bar
```
I've entered `eslint test.js`, but I get no error.

I've also tried `eslint -f html test.js > report.html` but the report indicates: "0 problems".

I am on Ubuntu 15.04 (64 bits). I have node 0.12.7 and npm 2.14.3.
The output of `which node npm eslint` is:
```
/usr/bin/node
/usr/bin/npm
/usr/bin/eslint
```

Best regards,
Olivier.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by ilyavolodin** [comment]
Can you provide a copy of your `.eslintrc` file please?

**Comment by obrac** [comment]
I have no `.eslintrc` file. To double check, I have used the commands with the `--no-eslintrc` option and I get the same results: no error printed and an html report with "0 problems".

**Comment by ilyavolodin** [comment]
In v1.0.0 we made a breaking change to ESLint, now ESLint doesn't not have rules turn on by default. So if you don't have .eslintrc file in your project, no rules are applied to your code. You should run `eslint --init` command to create configuration file for your project and then try running ESLint again.

**Comment by gyandeeps** [comment]
Eslint by default doesnt have any rules on. So you have to activate appropriate rules to show errors (if present).

Based on your example code above, you activate `semi: [2, "always"]` the  you would see one error.
To further play with eslint, please use are online demo.: http://eslint.org/demo/

**Comment by obrac** [comment]
I do I misses that! Sorry.

**Comment by obrac** [comment]
So, as a remainder:

- In [http://eslint.org/docs/rules/](http://eslint.org/docs/rules/), the first paragraph "rules" states: "none are enabled by default".
- In [http://eslint.org/docs/user-guide/migrating-to-1.0.0](http://eslint.org/docs/user-guide/migrating-to-1.0.0), the title of the first paragraph is: "All Rules Off by Default". It's a very informative document.

Of course, both pages are clearly visible from the [home page of eslint](http://eslint.org/), in the "User guide" item of the menu at the top of the page.

Thanks for your help, ilyavolodin and gyandeeps.

Best regards,
Olivier.
